src/components/LicenseRemindModal/index.jsx

```diff
@@ -0,0 +1,249 @@
+    } = this.props;
+    // Validate form data
+    this.validateFormData(formData);
+    // Configure the options to send to the assignment reminder API endpoint
+    const options = {
+      template: formData['email-template-body'],
+      template_greeting: formData['email-template-greeting'],
+      template_closing: formData['email-template-closing'],
+    };
+
+    if (isBulkRemind) {
+      options.bulkRemind = true;
+    } else if (!isBulkRemind && user) {
+      options.email = user.emailAddress;
+    }
+
+    return sendLicenseReminder(options).then((response) => {
+      this.props.onSuccess(response);
+    })
+      .catch((error) => {
+        throw new SubmissionError({
+          _error: [error.message],
+        });
+      });
+    /* eslint-enable no-underscore-dangle */
+  }
+
+  renderBody() {
+    const {
+      isBulkRemind,
+      submitFailed,
+      user,
+      pendingUsersCount,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        {submitFailed && this.renderErrorMessage()}
+        <div className="assignment-details mb-4">
+          <React.Fragment>
+            {isBulkRemind ?
+              <p className="bulk-selected-codes">Unredeemed Licenses: {pendingUsersCount}</p> :
+              <p className="bulk-selected-codes">Email: {user.emailAddress}
+              </p>}
+          </React.Fragment>
+        </div>
+        <form onSubmit={e => e.preventDefault()}>
+          <div className="mt-4">
+            <H3>Email Template</H3>
+            <Field
+              id="email-template-greeting"
+              name="email-template-greeting"
+              component={TextAreaAutoSize}
+              label="Customize Greeting"
+            />
+            <Field
+              id="email-template-body"
+              name="email-template-body"
+              component={TextAreaAutoSize}
+              label="Body"
+              disabled
+            />
+            <Field
+              id="email-template-closing"
+              name="email-template-closing"
+              component={TextAreaAutoSize}
+              label="Customize Closing"
+            />
+          </div>
+        </form>
+      </React.Fragment>
+    );
+  }
+
+  renderErrorMessage() {
+    const { error } = this.props;
+
+    return (
+      <div
+        ref={this.errorMessageRef}
+        tabIndex="-1"
+      >
+        <StatusAlert
+          alertType="danger"
+          iconClassName="fa fa-times-circle"
+          title="Unable to send reminder email"
+          message={error.length > 1 ? (
+            <ul className="m-0 pl-4">
+              {error.map(message => <li key={message}>{message}</li>)}
+            </ul>
+                    ) : (
+                        error[0]
+                    )}
+        />
+      </div>
+    );
+  }
+  renderTitle() {
+    const { title } = this.props;
+    return (
+      <React.Fragment>
+        <span className="d-block">{title}</span>
+        <small>
+          subtitle
```

Comment:
I think you should accept this in props too. And in the props either pass some subtitle string or an empty string if nothing fits the scenario.